devlib/utils/rendering.py

```diff
@@ -205,7 +206,11 @@ def _process_raw_file(self, fh):
                 for line in fh:
                     if line.startswith('---PROFILEDATA---'):
                         break
-                    self.frames.append(map(int, line.strip().split(',')[:-1]))  # has a trailing ','
+                    entries = map(int, line.strip().split(',')[:-1])  # has a trailing ','
+                    if entries[-1] <= last_vsync:
```

Comment:
Ah, the reason I only saw 7 frames is because this assumes the vsync timestamp is the last column; it's actually `entries[1]`. Fixing that gets us all the entries from `frames.csv.raw` in `frames.csv`, but there are still not as many as expected. Will keep investigating.